pkgs/top-level/default.nix

```diff
@@ -57,8 +57,8 @@ let
   platform = if platform_ != null then platform_
     else config.platform or platformAuto;
 
-  topLevelArguments = {
-    inherit system bootStdenv noSysDirs config crossSystem platform lib;
```

Comment:
Not sure why you changed the name, but it looks like you add lib later in all cases, or am I missing something?